apps/CatalystApp/Podfile.lock

```diff
@@ -1,7 +1,7 @@
 PODS:
-  - Mux-Stats-AVPlayer (3.2.0):
-    - Mux-Stats-Core (~> 4.4)
-  - Mux-Stats-Core (4.4.2)
+  - Mux-Stats-AVPlayer (3.3.2):
```

Comment:
question(non-blocking): Are the multiple version # jumps here because this was out of date?